### PR TITLE
Enhanced the customisability of Spinner component

### LINKED ISSUE #1873: Enhance the customisability of Spinner component
Part of https://github.com/bigbinary/neeto-ui/issues/1854

@goutham-subramanyam _a

cc: @praveen-murali-ind

### PR DESCRIPTION
**Pull request by goutham-subramanyam**
- Fixes #1873

**Description**

**Checklist**

- [ ] ~I have made corresponding changes to the documentation.~
- [ ] ~I have updated the types definition of modified exports.~
- [ ] ~I have verified the functionality in some of the neeto web-apps.~
- [ ] ~I have added tests that prove my fix is effective or that my feature works~
- [ ] ~I have added the necessary label (patch/minor/major - If package publish
      is required).~

**Reviewers**
@praveen-murali-ind _a Please review.

### CODE REVIEW

File: src/components/Spinner.jsx [hunk 1]
```diff
@@ -3,40 +3,37 @@ import React from "react";
...
-      data-testid="spinner"
-    >
-      <i className={spinnerItemClassName} />
-      <i className={spinnerItemClassName} />
-      <i className={spinnerItemClassName} />
-      <i className={spinnerItemClassName} />
-    </span>
-  );
-};
+const Spinner = ({ theme = "dark", size = 20, className = "" }) => (
```
Comment (praveen-murali-ind) [inline src/components/Spinner.jsx:8]: @goutham-subramanyam _a Let's restrict the size of the spinner component to two variants for now: `medium` size and `small`. Medium size should be the default size (`20px`). For small size variants, let's use `14px` for --neeto-ui-spinner-size.

File: src/components/Spinner.jsx [hunk 2]
```diff
@@ -3,40 +3,37 @@ import React from "react";
...
 
 Spinner.propTypes = {
   /**
    * To specify the theme of the Spinner.
    */
   theme: PropTypes.oneOf(Object.values(SPINNER_THEMES)),
+  /**
+   * To override the size of the spinner
+   */
+  size: PropTypes.number,
```
Comment (praveen-murali-ind) [inline src/components/Spinner.jsx:36]: @goutham-subramanyam _a Size should accept 'medium' and 'small'

**Review by praveen-murali-ind — COMMENTED** [review]
@goutham-subramanyam _a A few minor changes. The rest looks good to me.

**Comment by goutham-subramanyam** [comment]
@praveen-murali-ind _a Updated. Please check.

Screenshot 2023-10-23 at 12 50 08 PM

**Review by praveen-murali-ind — APPROVED** [review]

**Pull request merged**